.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
create a `CHANGELOG.md` Follow this model https://github.com/vtex-apps/store-graphql/blob/master/CHANGELOG.md